backend/src/routes/api/notebooks/notebookUtils.ts

```diff
@@ -121,10 +121,18 @@ export const postNotebook = async (
       throw error;
     });
 
+  let notebookLink:String = '';
+
+  if ((notebookData?.metadata?.annotations['opendatahub.io/notebook-type']=='code-server')||(notebookData?.metadata?.annotations['opendatahub.io/notebook-type']=='rstudio')) {
+    notebookLink = `https://${route.spec.host}/`
+  } else {
+    notebookLink = `https://${route.spec.host}/notebook/${namespace}/${notebookName}`
```

Comment:
The reason behind this path is to support kubeflow notebook controller culling: https://github.com/opendatahub-io/odh-manifests/tree/master/odh-notebook-controller#culling-endpoint

Do `code-server` or `rstudio` support culling?

Reply:
Ah, got it. And ttbomk and a quick search I did, no, there is no culling for code-server and rstudio. Well, there is for Rstudio Server, but it will simply hibernate a user's sessions, as it would do in a multi-user environment. But it does not allow you to simply kill the pod.
So I guess we'll have to stay with this different route scheme for now at least, and warn people there is no culling with  images others than Jupyter (which is the same for Kubeflow anyway as the "liveness" probe is hard-coded in the controller).